src/Quartz.HttpClient/HttpClient/HttpClientException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Quartz.HttpClient;
+
+public class HttpClientException : SchedulerException
```

Comment:
probably should be sealed and have some extra constructors, does this need to be public?